docs/node-operators/how-to/maintain-multisig-contract.md

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_position: 10
+title: Maintaining the Multisig Contract
+sidebar_label: Maintaining Multisig Contract
+hide_title: false
+vega_network: TESTNET
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+The [Multisig Control](../../api/bridge/interfaces/IMultisigControl.md) contract contains a set of signers whose signatures are required to complete actions on the Ethereum Bridge. The set of signers on the contract should match the set of consensus validators in the network. As validators join or leave the network and the set of consensus validators on the Vega network changes, the signer set on the Multisig Control contract must be updated to alongside those changes to maintain the security of the bridge.
+
+## Identifying when to update the contract
+
+The consensus set of validators on the Vega network can change because of the following:
+* The amounts staked to validators change such that a consensus validator is demoted and replaced by a standby validator with more stake
+* A consensus validator is showing poor performance and they are demoted and replaced by a more performant standby validator
+* The value of the network parameter <NetworkParameter frontMatter={frontMatter} param="network.validators.tendermint.number" hideValue={true} /> changes causing the size of the consensus set to increase or decrease
+
+Changes to the validator set only occur at the end of an epoch. Whether a node has moved into or out of the consensus set can be tracked using the [List Nodes](../../api/rest/data-v2/trading-data-service-list-nodes.api.mdx) API. The values of a node's `rankingScore.status` and `rankingScore.previousStatus` indicates whether it has been promoted or demoted, and what set it is now part of.
+
+
+A validator with the following ranking score values will have been *promoted* to a consensus validator and will need to be *added* to the multisig control
+```
+rankingScore.status == VALIDATOR_NODE_STATUS_TENDERMINT
+rankingScore.previousStatus != VALIDATOR_NODE_STATUS_TENDERMINT
+```
+
+A validator with the following ranking score values will have been *demoted* to a consensus validator and will need to be *removed* to the multisig control
+```
+rankingScore.status != VALIDATOR_NODE_STATUS_TENDERMINT
+rankingScore.previousStatus == VALIDATOR_NODE_STATUS_TENDERMINT
+```
+
+Seeing any validator with the above combination of ranking-score statuses means that action must be taken to prevent rewards from being withheld by the network. If the contract is still not updated and further changes to the consensus set occur then identifying the nodes to add or remove becomes harder.
+
+## How to update the contract
+
+To update the multisig contract by adding or removing a validator, there are two steps:
+- Send an `IssueSignatures` transaction into the Vega network to prompt it to issue signatures to add/remove a signer. This signature bundle can be retrieve from a data node
```

Comment:
```suggestion
- Send an `IssueSignatures` transaction into the Vega network to prompt it to issue signatures to add/remove a signer. This signature bundle can be retrieved from a data node
```

Reply:
Worth linking to the API that does this, if possible?

Reply:
Typo fixed, and links to the API are in the new "how to submit to ethereum without the multisig-tool" section.